internal/handlers/pg/pgdb/indexes.go

```diff
@@ -29,7 +32,7 @@ import (
 type Index struct {
 	Name   string
 	Key    IndexKey
-	Unique bool
+	Unique any
```

Comment:
Why is it no longer bool? Also what does `unique = types.NullType{}` represent in our metaIndex? A comment / doc string is appreciated here :hugs: 

If we want to specify unique, not unique and unset, then perhaps `*bool` is more suitable :thinking:

Reply:
I see it now that unset `unique` checks that `_id` cannot be unique.